packages/dapp/pages/index.tsx

```diff
@@ -10,7 +10,7 @@ import MigrateButton from "@/components/redeem/MigrateButton";
 import WalletNotConnected from "@/components/ui/WalletNotConnected";
 
 const index: FC = (): JSX.Element => {
-  const [twapPrice, setTwapPrice] = useState<ethers.BigNumber | null>(null);
+  const [, setTwapPrice] = useState<ethers.BigNumber | null>(null);
```

Comment:
I wonder if its a bad idea to remove these. I actually am unsure and have no strong opinion either way.

Reply:
we can do things like comment the variables out using sub-line block quotes.